app/screens/ScheduleScreen/ScheduleCard.tsx

```diff
@@ -29,14 +31,40 @@ const Header = ({ formattedEndTime, formattedStartTime, title, isPast }: HeaderP
   </View>
 )
 
-const Footer = ({ heading, subheading, isPast }: FooterProps) => (
-  <View style={$footerContainer}>
-    <Text preset="cardFooterHeading" style={isPast ? $pastFooterHeading : $footerHeading}>
-      {heading}
-    </Text>
-    <Text style={isPast ? $pastFooterSubheading : $footerSubheading}>{subheading}</Text>
-  </View>
-)
+const Footer = ({ heading, subheading, isPast, talkUrl, variant }: FooterProps) => {
+  return (
+    <View style={$footerContainer}>
+      <Text preset="cardFooterHeading" style={isPast ? $pastFooterHeading : $footerHeading}>
+        {heading}
+      </Text>
+      {isPast ? (
+        <>
+          <Text style={$pastFooterSubheading}>{subheading}</Text>
+          {
+            {
+              talk: (
```

Comment:
This syntax with the inline object is a bit hard to read. Can you move into a const outside of the render block?